### ISSUE TITLE
Make paths for "generated" and "l10n" folders configurable

### ISSUE THREAD

**Issue by LeviHassel** [localizely/flutter-intl-vscode#9]
It would be awesome if this extension allowed configurable paths and names for the `generated` and `l10n` folders. This would allow users to organize these folders within their existing file hierarchy. I think this would be most helpful placed alongside the other configuration options in `pubspec.yaml`.

For example, you could have:
```
flutter_intl:
  enabled: true
  l10n_path: lib/data/i18n/localizations
  generated_path: lib/data/i18n/generated
```

**Comment by aleksakrstic** [comment]
Hi @LeviHassel, I agree.
It is already in the extension roadmap.

Thanks for reporting this, it helps us to prioritize our work.

**Comment by DFelten** [comment]
That's currently the only thing that bothers me. It would be great if we could define where the folders are generated.

**Comment by marcos930807** [comment]
Any news on This??

**Comment by aleksakrstic** [comment]
Probably for the next major release

**Comment by aleksakrstic** [comment]
Configurable path to ARB folder is released with IDE extension `1.11.0`.

    flutter_intl:
      ...
      arb_dir: lib/l10n # Optional. Sets the directory of your ARB resource files. Provided value should be a valid path on your system. Default: lib/l10n

Will leave this issue open till path to generated folder is also configurable.

**Comment by aleksakrstic** [comment]
Generated folder is also configurable with IDE extension `1.13.0`.

    flutter_intl:
      ...
      output_dir: lib/generated # Optional. Sets the directory of generated localization files. Provided value should be a valid path on your system. Default: lib/generated
